Lib/gftools/builder/recipeproviders/googlefonts.py

```diff
@@ -268,6 +282,18 @@ def build_STAT(self):
                 build_stat_step["needs"] = other_variables
             self.recipe[last_target].append(build_stat_step)
 
+    def build_Avar2(self):
+        all_variables = [x for x in self.recipe.keys() if x.endswith("ttf")]
+        if len(all_variables) > 0:
+            last_target = all_variables[-1]
+            args = {"args": self.avar2file.name}
+            build_avar2_step = {
+                "postprocess": "buildAvar2",
+                **args,
+            }
+            self.recipe[last_target].append(build_avar2_step)
```

Comment:
This will only work when there is one VF produced by the builder (i.e. a Regular, but not Regular/Italic VFs). This is because the STAT step applies across all VFs produced, whereas this one works on a per-VF basis. You probably want to call it in the `build_a_variable` step instead of in the `build_all_variables` step, and do it to an individual VF rather the last one.

Reply:
Thanks. I've struggled to implement your feedback in `build_a_variable` since the func's target is a temp file. `gftools gen-avar2` uses the font's filename in order to fetch the relevant mapping. Since `gftools gen-avar2` allows multiple ttfs, I'm still able to call it from `build_all_variables` and it will process all the ttfs. Hopefully my changes should be better than this explanation,

Reply:
I've already tested it on a family with both Roman and Italic sources and it works.